packages/did-auth-siop-op-authenticator/src/utils/index.ts

```diff
@@ -0,0 +1,51 @@
+import { IParsedDID } from '../types/IDidAuthSiopOpAuthenticator';
```

Comment:
I think it might make sense to move this to the core plugin

Reply:
moved it to the core plugin